src/avfoundation.cs

```diff
@@ -2677,8 +2969,15 @@ interface AVFragmentedAsset_AVFragmentedAssetTrackInspection {
 		[Export ("tracksWithMediaType:")]
 		AVFragmentedAssetTrack [] GetTracks (string mediaType);
 
+		[Wrap ("This.GetTracks (mediaType.GetConstant ())")]
+		AVFragmentedAssetTrack [] GetTracks (AVMediaTypes mediaType);
+
 		[Export ("tracksWithMediaCharacteristic:")]
 		AVFragmentedAssetTrack [] GetTracksWithMediaCharacteristic (string mediaCharacteristic);
+
+		[Wrap ("This.GetTracksWithMediaCharacteristic (mediaCharacteristic.GetConstant ())")]
+		AVFragmentedAssetTrack [] GetTracksWithMediaCharacteristic (AVMediaCharacteristics mediaCharacteristic);
```

Comment:
Agreed, but we might one to set a few of them obsolete and encourage the enum usage. since we would have a GetTracks (string mediaType) that accidentally could be used with a mediaCharasteristic rather than a type.